styx-api-service/src/test/java/com/spotify/styx/api/WorkflowResourceTest.java

```diff
@@ -99,9 +118,9 @@ public WorkflowResourceTest(Api.Version version) {
 
   private static final ByteString STATEPAYLOAD_FULL =
       ByteString.encodeUtf8("{\"enabled\":\"true\", \"docker_image\":\"cherry:image\", "
-                            + "\"commit_sha\":\"" + VALID_SHA
-                            + "\", \"next_natural_trigger\":\"2016-08-10T07:00:01Z\", "
-                            + "\"next_natural_offset_trigger\":\"2016-08-10T08:00:01Z\"}");
+          + "\"commit_sha\":\"" + VALID_SHA
```

Comment:
This seems not aligned with default formatter. Also applies to other format related changes.